src/base/bittorrent/private/filterparserthread.cpp

```diff
@@ -238,15 +249,19 @@ int FilterParserThread::parseDATFilterFile()
                 ++ruleCount;
             }
             catch (std::exception &e) {
-                LogMsg(tr("IP filter exception thrown for line %1. Exception is: %2").arg(nbLine)
-                                               .arg(QString::fromLocal8Bit(e.what())), Log::CRITICAL);
+                ++parseErrorCount;
+                addLog(tr("IP filter exception thrown for line %1. Exception is: %2")
+                       .arg(nbLine).arg(QString::fromLocal8Bit(e.what())));
             }
         }
 
         if (start >= dataSize)
             offset = 0;
     }
 
+    if (parseErrorCount >= MAX_LOG_LINES)
+        LogMsg(tr("Extra %1 IP filter parsing errors occurred.", "Extra 513 IP filter parsing errors occurred.")
+               .arg(parseErrorCount - MAX_LOG_LINES + 1), Log::CRITICAL);
```

Comment:
Can't understand this math. Test: MAX_LOG_LINES = 0, parseErrorCount = 1. This would print "Extra 2 … errors…" while it has to print "Extra 1 …".